packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/include/camera_avfoundation/FLTCam.h

```diff
@@ -113,7 +113,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)startImageStreamWithMessenger:(NSObject<FlutterBinaryMessenger> *)messenger;
 - (void)stopImageStream;
 - (void)setZoomLevel:(CGFloat)zoom withCompletion:(void (^)(FlutterError *_Nullable))completion;
-- (void)setUpCaptureSessionForAudio;
+- (void)setUpCaptureSessionForAudioIfNeeded;
```

Comment:
Is this called outside the implementation? If not it should be moved to the private category in the `.m` file.

Reply:
It is also called from `prepareForVideoRecordingWithCompletion` in CameraPlugin.m.